### DISCUSSION TITLE
Unable to Materizlize manhattan_map asset in Dagster Essentials Course

### DISCUSSION THREAD

**Discussion by mtopolskis** [dagster-io/dagster#26234] [Q&A] [answered]
Hello. I am am having difficulty with the Dagster Essentials Course - [https://courses.dagster.io/courses/dagster-essentials](url) 

I have installed all components, and completed all previous tasks up to Lesson 4 Step 5.

I am stuck on materializing the manhattan_map asset. When I attempt to materialize it, the run starts without error, but it continues indefinitely (I've let it run for around 2 hours before I stepped away and came back to a timeout. 
I am running Python in VS code

Before I list my troubleshooting steps and configs, I wanted to reach out and see if there is maybe an issue with this course or this asset specifically and if anyone else has either encountered this issue, or was able to successfully materialize the asset. 

Is there a known issue with this course, or am I doing something wrong?

**Comment by memke1** [comment, marked as answer]
Hi there, I ran into the same issue and finally managed to fix it. It has to do with the kaleido version 0.2.1 in combination with using windows 10 enterprise. 
Simply open command terminal and run the following command to fix it, worked like a charm for me:

`pip install --upgrade "kaleido==0.1.*"`

**Reply by guykatz** [reply]
that did the trick for me too when facing this issue so thanks.
my only concern is that I could not undetrsand why it happened, there were no logs related to this that I could see.
how did you eventually undertsdand what the issue was?
thanks!

**Reply by memke1** [reply]
I went on debugging the asset manually line-by-line, then figured it had to do something with plotly - I also had no logs because there was no fail while executing, code was just running forever. 
I attempted to plot something very simple using plotly and ran into the same issue, then found above fix on stackoverflow.

**Reply by guykatz** [reply]
you saved a lot of time for me so thanks :)

**Reply by cmpadden** [reply]
Thank you @memke1 - we've pinned the version in the university project. For those who created their project before this, they'll have to update it manually, like you've said.
